### ISSUE TITLE
Linux deb package version number not the same as sbtVersion value  

### ISSUE THREAD

**Issue by hmf** [sbt/sbt#4342]
## steps

command 1: sbt sbtVersion
command 2. sudo apt-get install sbt

## problem

command 1: show the version number "1.1.6"
command 2:  "sbt is already the newest version (1.2.1)."

## expectation

Expected the same version numbers to be the same.

## notes

sbt version: *1.1.6*

**Comment by eed3si9n** [comment]
The version of the sbt's main version is determined by `project/build.properties` file.

**Comment by hmf** [comment]
Apologies for insisting but I don't understand. Just to confirm:

* The latest version is 1.2.1 which is set by  `project/build.properties`
* If I install the latest version and execute `sbt sbtVersion` I should get 1.2.1
* I assume the latest deb package has sbt version 1.2.1

So why do I get 1.1.6 from the sbt installed via the deb package?
 
TIA

**Comment by eed3si9n** [comment]
Which Ubuntu version are you using? Do you see any error message when you go through the steps in https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html?

**Comment by hmf** [comment]
Sorry for the late reply. 

I had followed those instructions without a hitch. Just to be on the safe side I did an `update` and `upgrade`. I also did a new `sbt sbtVersion` and got the expected `1.2.1`.  I now realize what I did wrong. I had previously executed the latter command in a project directory with a build file that explicitly asks for 1.1.6. My mistake!

When you referred to the `project/build.properties` file, I assumed you were referring to SBT's project. My sincere apologizes for taking up your time.
